apps/frontend/components/FormValidation/DroughtFormValidation/DroughtFormValidation.tsx

```diff
@@ -49,8 +46,8 @@ export const DroughtFormValidation = ({
         district: formattedForm.district,
         commune: formattedForm.commune,
       },
-      interviewer: formattedForm.reportName,
-      disTyp: formattedForm.disasterType,
+      interviewer: formattedForm.entryName,
```

Comment:
What does interviewer mean in this context?

Reply:
it is the name of the name of the person that filled out the form. It is something we can change